samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+.. _crypto_test:
+
+Cryptography test sample
+########################
+
+The cryptography test sample verifies the functionality of the nrf_crypto operations by using known test vectors approved by the National Institute of Standards and Technology (NIST) and others.
+All available nrf_crypto backends can be tested by modifying the nrf_crypto section in the sdk_config file, as described in lib_crypto_config.
+
```

Comment:
Additional paragraph needed to emphasize that we are basing this on ztest

"The crypto test application utilizes ztest to execute the tests. This includes mechanisms of test reporting in RTT output in a specific format. Please see the chapter 'Building and running' for details."

A link towards the chapter "Buidling and running" as well as a link to "ztest" (documented in Zephyr scope) is appreciated

Reply:
done